src/ci/bin/test.inc.sh

```diff
@@ -1658,37 +1631,6 @@ cromwell::build::check_published_artifacts() {
     fi
 }
 
-cromwell::build::publish_artifacts() {
-    if [[ "${CROMWELL_BUILD_PROVIDER}" == "${CROMWELL_BUILD_PROVIDER_TRAVIS}" ]] && \
-        [[ "${CROMWELL_BUILD_TYPE}" == "sbt" ]] && \
-        [[ "${CROMWELL_BUILD_SBT_INCLUDE}" == "" ]] && \
-        [[ "${CROMWELL_BUILD_EVENT}" == "push" ]]; then
-
-        if [[ "${CROMWELL_BUILD_BRANCH}" == "develop" ]]; then
-            # Publish images for both the "cromwell develop branch" and the "cromwell dev environment".
-            CROMWELL_SBT_DOCKER_TAGS=develop,dev \
-                cromwell::private::publish_artifacts_and_docker \
```

Comment:
This private function appears to be no longer called after this removal, so should it also be removed?